class-events-manager-securesubmit.php

```diff
@@ -587,9 +589,11 @@ public function setConfig()
     {
         $config = new ServicesConfig();
         $config->secretApiKey = get_option('em_'.$this->gateway.'_secret_key');
-        $config->serviceUrl = "https://cert.api2.heartlandportico.com";
+        $env = $config->environment;
```

Comment:
The SDK isn't currently setting the environment for you in the version bundled as a part of this PR, so you will need to determine the environment, probably based off of the secret API key. As is, this will only ever be TEST

Reply:
updated